Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "AWSS3"
 uuid = "1c724243-ef5b-51ab-93f4-b0a88ac62a95"
-version = "0.8.2"
+version = "0.8.3"
```

Comment:
GH is not allowing me to comment further down in the file.
However, do we not need to bump:

```toml
[compat]
AWS = "1.25"
```

Reply:
done